ngraph/frontend/tensorflow/src/ngraph_conversions.hpp

```diff
@@ -56,7 +54,7 @@ void Transpose3D(std::shared_ptr<ov::Node>& node) {
 
```

Comment:
`reshaped_shape` is not needed. 
please compute a rank as `rank = node.get_rank().length()` but please check that rank is static first

Reply:
.get_shape and reshaped_shape are not used and should be deleted
propose to fix it in the next PR